agent/agents/postgres/pgstatmonitor/pgstatmonitor_models.go

```diff
@@ -177,15 +177,15 @@ func NewPgStatMonitorStructs(v pgStatMonitorVersion) (*pgStatMonitor, reform.Vie
 			field{info: parse.FieldInfo{Name: "WalBytes", Type: "int64", Column: "wal_bytes"}, pointer: &s.WalBytes})
 	}
 
-	if v <= pgStatMonitorVersion10PG12 {
```

Comment:
This is actual fix, columns are not related to PGSM and PG version, but only to PG version.